### DISCUSSION TITLE
Drop event not cancelled when dropping out of inventory instead of pressing q while holding (creative)

### DISCUSSION THREAD

**Discussion by Tricked-dev** [Minestom/Minestom#1970] [Help] [answered]
https://github.com/Minestom/Minestom/assets/72335827/7c98832f-323c-40b0-a306-c45e74a8e28d

works in survival but not in creative

```kotlin
        .addListener(ItemDropEvent::class.java) {event:ItemDropEvent ->
          event.isCancelled = true;
        }
```

Is this a bug or am i doing something wrong?

**Comment by DevBlokJochem** [comment, marked as answer]
Hey, I think this is a bug. For now, you can listen to the inventory click event.

**Reply by Tricked-dev** [reply]
ah thanks!
